arch/mips/src/pic32mz/pic32mz_ethernet.c

```diff
@@ -254,7 +254,7 @@
  * header
  */
 
-#define BUF ((struct eth_hdr_s *)priv->pd_dev.d_buf)
+#define BUF ((FAR struct eth_hdr_s *)priv->pd_dev.d_buf)
```

Comment:
remove

Reply:
Currently `FAR` is heavily used in arch/mips files

Reply:
Ok, let's keep it